src/core/include/cesium/omniverse/OmniTileset.h

```diff
@@ -11,14 +11,12 @@
 #include <vector>
 
 namespace Cesium3DTilesSelection {
-struct ImageCesium;
```

Comment:
Is this removed because `ImageCesium` is available elsewhere?

Reply:
Yeah, other files are including / forward declaring `ImageCesium`. At one point `OmniTileset.h` must have been using it.